### DISCUSSION TITLE
Custom css override possible in NEXT.js

### DISCUSSION THREAD

**Discussion by kwangjaeyoo** [wojtekmaj/react-calendar#870] [Q&A]
### Before you start - checklist

- [X] I followed instructions in documentation written for my React-Calendar version
- [X] I have checked if this bug is not already reported

### Description

Is it possible to override the css module in next.js?
react app is working but nextjs not working 

======================
calender custom component 

import 'react-calendar/dist/Calendar.css'
import './Calendar.module.css'

import Calendar from 'react-calendar'

const CcCalendar = () => {
  return (
    <div>
      <Calendar
        onClickDay={(value) => console.log('TODO ' + value)}
        formatDay={(locale, date) =>
          date.toLocaleString('en', { day: 'numeric' })
        }
        // minDetail="month" // 상단 네비게이션에서 '월' 단위만 보이게 설정
        // maxDetail="month" // 상단 네비게이션에서 '월' 단위만 보이게 설정
        nextLabel={<>TODO</>}
        prevLabel={<>TODO</>}
        next2Label={null}
        prev2Label={null}
        calendarType={'US'} // 일요일 시작
      />
    </div>
  )
}

export default CcCalendar

====================== 
Calendar.module.css

.react-calendar__month-view__days__day--weekend {
  color: #aaffff;
}
... some more...
======================

I have the same problem with that link

https://stackoverflow.com/questions/74829545/react-calendar-css-overriding-isnt-applying-to-the-calendar

### Steps to reproduce

............

### Expected behavior

............

### Actual behavior

............

### Additional information

............

### Environment

   "react": "^18.2.0",
    "react-calendar": "^4.3.0",
    "next": "^13.3.0",

**Comment by wojtekmaj** [comment]
You could always copy the base CSS and simply edit it to your liking, instead of having all the overrides.
